File: src/main/java/by/architecture/map/repository/ArchitecturalStyleRepository.java
package by.architecture.map.repository;

import by.architecture.map.entity.ArchitecturalStyle;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ArchitecturalStyleRepository extends JpaRepository<ArchitecturalStyle, Integer> {

    boolean existsByName(String name);
}

File: src/main/kotlin/by/architecture_map/belarus/repository/jpa/ArchitecturalStyleRepository.kt
package by.architecture_map.belarus.repository.jpa

import by.architecture_map.belarus.entity.ArchitecturalStyle
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ArchitecturalStyleRepository : JpaRepository<ArchitecturalStyle, Int>